Maestro/subscriptions.json

```diff
@@ -55,7 +55,7 @@
             "-GitHubUser dotnet-bot",
             "-GitHubEmail dotnet-bot@microsoft.com",
             "-GitHubPassword `$(`$Secrets['DotNetBotGitHubPassword'])",
-            "-CoreFxVersionUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/release/1.0.0/Latest.txt",
+            "-VersionFileUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/release/1.0.0/Latest.txt",
```

Comment:
Actually the release branch still has the old argument name for now so you sholudn't change it yet.